imports/plugins/included/shippo/server/methods/shippo.js

```diff
@@ -409,10 +409,9 @@ export const methods = {
       const shippoAddressFrom = createShippoAddress(shop.addressBook[0], shop.emails[0].address, purpose);
       // product in the cart has to have parcel property with the dimensions
       if (cart.items && cart.items[0] && cart.items[0].parcel) {
-        const unitOfMeasure = shop && shop.unitsOfMeasure && shop.unitsOfMeasure[0].uom || "KG";
-        // at the moment shops don't have a kind of unitOfMeasure for distance
-        // so we put CM...
-        shippoParcel = createShippoParcel(cart.items[0].parcel, unitOfMeasure, "CM");
+        const unitOfMeasure = shop && shop.baseUOM || "kg";
+        const unitOfLength = shop && shop.baseUOL || "cm";
```

Comment:
A little weird that the fallback units are different than the default base units, but I don't see any real problem with it and cm/kg are more widely used.